gulpfile.js

```diff
@@ -237,13 +238,17 @@ gulp.task('clean', function(cb) {
   ], cb);
 });
 
-gulp.task('start-foxbox-simulator', function() {
+gulp.task('start-simulators', function() {
   foxboxSimulator = gls.new('tests/foxbox-simulator/http-server.js');
   foxboxSimulator.start();
+
+  registrationServerSimulator = gls.new('tests/registration-server-simulator/http-server.js');
```

Comment:
Following up on the live reload issue: I wonder if we shouldn't turn off live reload https://www.npmjs.com/package/gulp-live-server#glsargs-options-livereload